tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -22,35 +24,36 @@
     argnames=["interface_type", "pvc_size", "file_size"],
     argvalues=[
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '1', '600M'], marks=pytest.mark.polarion_id('OCS-2356')
+            *[constants.CEPHBLOCKPOOL, '1', '600Mi'], marks=pytest.mark.polarion_id('OCS-2356')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '25', '15GB'], marks=pytest.mark.polarion_id('OCS-2340')
+            *[constants.CEPHBLOCKPOOL, '25', '15Gi'], marks=pytest.mark.polarion_id('OCS-2340')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '50', '30GB'], marks=pytest.mark.polarion_id('OCS-2357')
+            *[constants.CEPHBLOCKPOOL, '50', '30Gi'], marks=pytest.mark.polarion_id('OCS-2357')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '100', '60GB'], marks=pytest.mark.polarion_id('OCS-2358')
+            *[constants.CEPHBLOCKPOOL, '100', '60Gi'], marks=pytest.mark.polarion_id('OCS-2358')
         ),
         pytest.param(
-            *[constants.CEPHFILESYSTEM, '1', '600M'], marks=pytest.mark.polarion_id('2341')
+            *[constants.CEPHFILESYSTEM, '1', '600Mi'], marks=pytest.mark.polarion_id('2341')
         ),
         pytest.param(
-            *[constants.CEPHFILESYSTEM, '25', '15GB'], marks=pytest.mark.polarion_id('2355')
+            *[constants.CEPHFILESYSTEM, '25', '15Mi'], marks=pytest.mark.polarion_id('2355')
```

Comment:
shouldn't be 15Gi ?

Reply:
@Avilir you are right! Fixed it, thank you!